website/src/jest/img/logos/shopify.svg

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- Generator: Adobe Illustrator 18.1.1, SVG Export Plug-In . SVG Version: 6.00 Build 0)  -->

```

Comment:
Aw, I just noticed that this isn’t optimized. For shame 😔

I’ll add one that’s lighter.